packages/frontend/src/app/create-store/page.tsx

```diff
@@ -117,6 +118,12 @@ const StoreCreation = () => {
         });
 
         if (transaction!.status == "success") {
+          //Add relay tokenId
+          const relayURL =
+            (process && process.env["RELAY_ENDPOINT"]) ||
+            "ws://localhost:4444/v2";
+          const relayEndpoint = await discoverRelay(relayURL);
```

Comment:
```suggestion
          // TODO: use process.env.NEXT_PUBLIC_RELAY_TOKEN_ID in prod
          const relayEndpoint = await discoverRelay(relayURL);
```


We should probably condense what i wrote here a bit more, since `discoverRelay` only works in testing.

https://github.com/masslbs/Tennessine/blob/90c10f0f0baa6c0ae5a01283ea4a26741ce3f437/packages/frontend/src/context/MyContext.tsx#L286-L292